sample/src/main/java/io/instaleap/statusprogressbar/Constants.kt

```diff
@@ -0,0 +1,14 @@
+package io.instaleap.statusprogressbar
+
+const val DEFAULT_ITEMS = 1
+const val ZERO_ITEMS = 0
+const val TOTAL_ITEMS_EXAMPLE = 10
+const val ONE_UNIT = 1
+const val ZERO_RANGE = 0
+const val TEN_RANGE = 10
+const val INIT_RANDOM_VALUE = 0
+const val END_RANDOM_VALUE = 100
+
+const val GREEN_COLOR_BAR = "#00D59E"
```

Comment:
Why do u prefer it as constant instead of refer to android resource?

Reply:
Library doesn't know about the resources that are been using in de app directly. :)